tests/tests/test_utils.py

```diff
@@ -117,4 +160,4 @@ def test_cannot_deserialize_non_exception(self) -> None:
         ]:
             with self.subTest(data):
                 with self.assertRaises((TypeError, ImportError)):
-                    utils.exception_from_dict(data)
+                    utils.exception_from_dict(data)  # type: ignore
```

Comment:
Suggestion: Remove the type ignore, and add an empty `exc_traceback` to the data fixtures.

Reply:
Not sure that will be sufficient for mypy, I might need to move the iterable into its own variable to declare the type explicitly.